src/Backend/Modules/Pages/Js/Pages.js

```diff
@@ -448,6 +450,16 @@ jsBackend.pages.extras =
 		}
 	},
 
+	// show the groups for authentication
+	showGroups: function(e)
+	{
+		// save element to variable
+		var element = $(this);
+		var list = $('#groupList');
```

Comment:
It would be safer to use something like `.js-authentication-groups'
That way if we do a rewrite we won't accidentally break the js since we know js is listening for something here